utils/src/main/kotlin/Extensions.kt

```diff
@@ -41,10 +41,9 @@ import java.nio.file.Path
 import java.nio.file.SimpleFileVisitor
 import java.nio.file.StandardCopyOption
 import java.nio.file.attribute.BasicFileAttributes
+import java.security.MessageDigest
```

Comment:
That change requires more explanation in the commit message, I guess it's something along the lines of "commons codec only provides a nicer interface for `MessageDigest`, but no additional functionality".

Reply:
I've added more explanatory text to the message of the commit before, as that's actually the first commit introducing this kind of change.